### PR TITLE
Added a confirm box on enabling the Unauthenticated ACF upload enablement

### PR DESCRIPTION
**Pull request by Renuka9527**
Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=397591

Description: Added confirm message before enabling the Unauthenticated ACF upload enablement
Screenshot:
image

### CODE REVIEW

File: src/locales/en-US.json [hunk 1]
```diff
@@ -1255,6 +1255,7 @@
       "successNetworkPolicyUpdate": "Successfully updated %{policy}.",
       "successNextBootToast": "Applying changes to %{policy}. Changes made to %{policy} will take effect on next reboot."
     },
+    "acfUploadEnablementConfirmText": "Enabling this setting will allow an unauthorised user to upload ACF  for infinite period.\n You really confirm to enable this setting  ?",
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: There are extra spaces in the text, Can those be removed?

Comment (Renuka9527) [reply src/locales/en-US.json]: @Nikhil-Ashoka  Addressed

File: src/locales/en-US.json [hunk 2]
```diff
@@ -1268,6 +1268,7 @@
       "successNetworkPolicyUpdate": "Successfully updated %{policy}.",
       "successNextBootToast": "Applying changes to %{policy}. Changes made to %{policy} will take effect on next reboot."
     },
+    "acfUploadEnablementConfirmText": "Caution: Enabling this setting allows unauthenticated users to upload Access Control Files (ACFs) without logging in and without pressing Function 74 on the Operator Panel.  Do you want to proceed?",
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: Please Remove the additional spaces here: (.  Do you want to proceed?)

**Review by Nikhil-Ashoka — APPROVED** [review]
Looks Good!!

**Pull request merged**